scripts/webpack.config.js

```diff
@@ -117,12 +122,7 @@ function getConfig(isReactExternalized) {
             }),
             new MiniCssExtractPlugin({
                 filename: '[name].css',
-            }),
-            new OptimizeCssAssetsPlugin({
-                cssProcessorOptions: {
-                    discardComments: { removeAll: true },
-                    parser: safeParser,
-                },
+                ignoreOrder: true,
```

Comment:
i have a feeling our throughout the repo CSS isn't structured consistently to be able to use this setting. is this needed?

Reply:
![image](https://github.com/box/box-ui-elements/assets/103291617/e81f9fc0-dcbd-4243-bf8a-c1396122e36d)
this comes up if we dont